tests/inject/dynamic/media-query.tests.ts

```diff
@@ -116,4 +116,24 @@ describe('MEDIA QUERIES', () => {
         expect(getComputedStyle(document.querySelector('h1')).backgroundColor).toBe('rgb(0, 102, 0)');
         expect((document.querySelector('.testcase-style').nextElementSibling as HTMLStyleElement).sheet.cssRules.length).toBe(2);
     });
+
+    it('should style print/media query', () => {
+        container.innerHTML = multiline(
+            '<style class="testcase-style">',
+            '    h1 { background: green; }',
+            '    h1 strong { color: orange; }',
+            '</style>',
+            '<style class="testcase-style-2" media="print, screen and (max-width: 9999999px)">',
+            '    h1 { background: gray; }',
+            '    h1 strong { color: red; }',
+            '</style>',
+            '<h1>Some test foor...... <strong>Oh uhm removing styles :(</strong>!</h1>',
+        );
+        debugger;
```

Comment:
debugger;

Reply:
Going to add an eslint rule later I forget this so many times within tests.